app/views/personal_details/create.html.erb

```diff
@@ -0,0 +1,20 @@
+<span class="govuk-caption-xl"><%= t('application_form.personal_details_section.heading') %></span>
+<h1 class="govuk-heading-xl"><%= t('application_form.review_answers') %></h1>
+
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <dl class="govuk-summary-list govuk-!-font-size-16 govuk-summary-list--no-border">
+      <%= render partial: 'shared/description_list_item', locals: { field: :title, value: @personal_details.title } %>
+      <%= render partial: 'shared/description_list_item', locals: { field: :first_name, value: @personal_details.first_name } %>
+      <%= render partial: 'shared/description_list_item', locals: { field: :last_name, value: @personal_details.last_name } %>
+      <%= render partial: 'shared/description_list_item', locals: { field: :preferred_name, value: @personal_details.preferred_name } %>
+      <%= render partial: 'shared/description_list_item', locals: { field: :date_of_birth, value: @personal_details.date_of_birth.to_s(:govuk_human_readable) } %>
+      <%= render partial: 'shared/description_list_item', locals: { field: :nationality, value: @personal_details.nationality } %>
+    </dl>
+  </div>
+</div>
+
+<%= link_to tt_application_path, role: 'button', draggable: false, class: 'govuk-button' do %>
```

Comment:
What's the impact of `draggable: false` here?

(It might have just been copy-pasted in, but I'm curious!)

Reply:
Good catch. No idea. I probably cargo-culted it from the prototype or the Service Manual. I now can't find it in either of those places. Let's lose it.

Reply:
Sorry, missed one of 'em - suggestion you can just click in at https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/83/files#r301541557, or you might prefer to rebase?